all_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+# Bazel tests
+bazel test //... --config=ci
+
+# Test targets that must fail
+tests/targets-that-must-fail/run-tests.sh
```

Comment:
Now that we have `all_tests.sh`, let's inline failure tests, rather than keeping the extra level of indirection.

Reply:
I was thinking about auto-discovering them using `bazel query`.

Reply:
Good idea. They'll need to be marked using the `manual` tag as well, so that `bazel test //...` doesn't fail. This way we don't need a separate WORKSPACE for them.

Reply:
Good point!

Reply:
The PR has been updated with:

- Auto detection of failure test
- The separate workspace was removed